DataManagementSystem/Service/IRODSStorageElementHandler.py

```diff
@@ -579,7 +577,7 @@ def export_getResourceInfo( self , resource = None ):
     """ Send the storage element resource information
 """
 
-    conn , error, userDict = self.__irodsClient( )
+    conn , error, _ = self.__irodsClient()
```

Comment:
Yes Yes, I know that. I just like *personally* to add something behind, but I was wondering if others like to follow that as well